packages/dev-utils/src/subproviders/empty_wallet_subprovider.ts

```diff
@@ -0,0 +1,23 @@
+import { JSONRPCRequestPayload } from 'ethereum-types';
+
+import { Callback, ErrorCallback } from './types';
+
+import { Subprovider } from './subprovider';
+
+/**
+ * This is duplicated from Subproviders
```

Comment:
why not import it from `subproviders`? that should be fine, right?

Reply:
cyclic dependency :(